Mod Files/data/hyperspace.xml

```diff
@@ -206,6 +206,10 @@ Tends to be helpful with "weirder" weapons in mods due to having more clear info
 
 <showReactor enabled="false"/>
 
+<!-- Shows additional value according to the selected ship in the hangar -->
+
+<showExtraShipInfo enabled="true" noMaxCrew="false"/>
```

Comment:
It feels weird for max crew to be the only one with separate option, and that you have to disable it instead of enabling it. Maybe this tag should be laid out more like this
```xml
<extraShipInfo missileCount="true" droneCount="true" maxCrew="true"/>
```
Where each one enables a particular element